docs/users-and-roles-swagger.yaml

```diff
@@ -204,6 +204,36 @@ paths:
         responses: {}
         httpMethod: POST
         type: aws_proxy
+  /external-client-userinfo:
```

Comment:
/external-client/userinfo ?

Reply:
yeah, was not sure about this. what you suggest is in conflict with /external-clients/{clientId}  in my head, how about  /clientinfo ?